### DISCUSSION TITLE
Plans to support React-Redux v8 and React v18

### DISCUSSION THREAD

**Discussion by andreykovalenko** [reduxjs/redux-toolkit#2190] [General]
Hey,
 
I could not find any info about the migration plans to the new libraries. It is failing when I try to update React-Redux and React libs.

It is hard coded in package.json
  ```
   "react-redux": "npm:7.2.4",
    "react": "npm:17.0.2",
    "react-dom": "npm:17.0.2"
```

Please let me know if there is a workaround or plans to support it soon?

Many Thanks

**Comment by markerikson** [comment]
Are you sure you're looking in the right spot?  Those are both explicitly listed as peer dependencies, and we just updated the React peer dep yesterday to list 18 as final:

https://github.com/reduxjs/redux-toolkit/blob/ae2a3c602241e8dc4b883c565f80b6a93ef653c4/packages/toolkit/package.json#L107-L109

**Reply by andreykovalenko** [reply]
thank you I was checking the root level package.json. It works!

**Reply by markerikson** [reply]
I'm actually really confused what was "failing" then :)  What were you trying, and what errors were you seeing?
